src/System.IO.Pipelines/ReadableBuffer.cs

```diff
@@ -40,7 +36,14 @@ public struct ReadableBuffer : ISequence<ReadOnlyMemory<byte>>
         /// <summary>
         /// The first <see cref="Memory{Byte}"/> in the <see cref="ReadableBuffer"/>.
         /// </summary>
-        public Memory<byte> First => _first;
+        public Memory<byte> First
```

Comment:
This made all of the code ugly. Filed this https://github.com/dotnet/corefxlab/issues/1285